src/lib/twitter_web/controllers/post_controller.ex

```diff
@@ -0,0 +1,68 @@
+defmodule TwitterWeb.PostController do
+  @moduledoc """
+  The Post Controller.
+  """
+  use TwitterWeb, :controller
+
+  alias Twitter.Tweets
+
+  @doc """
+  Creates a post and returns the homepage of the application
+  """
+  @spec create(Plug.Conn.t(), map() | any()) :: Plug.Conn.t()
+  def create(conn, params) do
+    current_user = conn.assigns.current_user
+    content = params["content"]
```

Comment:
```suggestion
  def create(conn, %{"content" => content}) do
    current_user = conn.assigns.current_user
```
Use pattern matching